pkg/tcl/expressionstcl/call.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2024 Testkube.
+//
+// Licensed as a Testkube Pro file under the Testkube Community
+// License (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//     https://github.com/kubeshop/testkube/blob/main/licenses/TCL.txt
+
+package expressionstcl
+
+import (
+	"fmt"
+	"maps"
+	"strings"
+)
+
+type call struct {
+	name string
+	args []Expression
+}
+
+func newCall(name string, args []Expression) Expression {
+	for i := range args {
+		if args[i] == nil {
+			args[i] = None
+		}
+	}
+	return &call{name: name, args: args}
+}
+
+func (s *call) WillBeString() bool {
+	return s.name == "string"
```

Comment:
should it be const?

Reply:
Thanks, good point.

This is kind-of small workaround, to simplify the generated schema for known stdlib function; after you've mentioned it, I noticed that there are more functions that will be always string, like `join`, `tojson` or `toyaml`.

Which option do you think will be better:
* add `var knownStringFunctions = []string{"string", "tojson"} // etc` on top of `stdlib.go` and use it
* adjust `stdFunctions` to be map[string]struct{willBeString: bool, handler: func(...StaticValue) (Expression, error)}
?

Reply:
second one looks more consistent to me.

Reply:
Pushed the change 👍

Added:
```go
type StdFunction struct {
	ReturnType string
	Handler    func(...StaticValue) (Expression, error)
}
```

And used it in `stdFunctions`

Reply:
To make it more flexible, I have also replaced `StringAwareExpression.WillBeString() bool` with `Expression.Type() Type`, so every expression may by itself inform about the result type. This way in the future we could apply better reflection - both for validation and error messages, as well as predictions and optimizations.